PHP7 interop tests are broken

Seems like it got broken by some cronet change?

```
[C]       Compiling src/core/ext/transport/cronet/transport/cronet_transport.c
src/core/ext/transport/cronet/transport/cronet_transport.c: In function 'execute_from_storage':
src/core/ext/transport/cronet/transport/cronet_transport.c:792:5: error: 'method' may be used uninitialized in this function [-Werror=maybe-uninitialized]
     cronet_bidirectional_stream_start(s->cbs, url, 0, method, &s->header_array,
     ^
src/core/ext/transport/cronet/transport/cronet_transport.c:784:17: note: 'method' was declared here
     const char *method;
                 ^
cc1: all warnings being treated as errors
make: *** [/var/local/git/grpc/objs/opt/src/core/ext/transport/cronet/transport/cronet_transport.o] Error 1
Makefile:2171: recipe for target '/var/local/git/grpc/objs/opt/src/core/ext/transport/cronet/transport/cronet_transport.o' failed
+ EXITCODE=2
+ docker rm -f build_grpc_interop_php7_10952541-b36c-453d-bd05-55a8095bea17
build_grpc_interop_php7_10952541-b36c-453d-bd05-55a8095bea17
+ exit 2

FAILED: build_docker_php7 [ret=2, pid=12498]
```

The performance tests also seem broken:
```
[LD]      Linking /home/jenkins/workspace/gRPC_performance_master/libs/opt/libgrpc_unsecure.so.1.1.0-dev
src/core/ext/transport/cronet/transport/cronet_transport.c: In function ‘execute_from_storage’:
src/core/ext/transport/cronet/transport/cronet_transport.c:792:5: error: ‘method’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
     cronet_bidirectional_stream_start(s->cbs, url, 0, method, &s->header_array,
     ^
src/core/ext/transport/cronet/transport/cronet_transport.c:784:17: note: ‘method’ was declared here
     const char *method;
                 ^
cc1: all warnings being treated as errors
Makefile:2171: recipe for target '/home/jenkins/workspace/gRPC_performance_master/objs/opt/src/core/ext/transport/cronet/transport/cronet_transport.o' failed
make: *** [/home/jenkins/workspace/gRPC_performance_master/objs/opt/src/core/ext/transport/cronet/transport/cronet_transport.o] Error 1
make: *** Waiting for unfinished jobs....

FAILED: make [ret=2, pid=2340]
```

--- a/src/core/ext/transport/cronet/transport/cronet_transport.c
+++ b/src/core/ext/transport/cronet/transport/cronet_transport.c
@@ -780,8 +780,8 @@ static enum e_op_result execute_stream_op(grpc_exec_ctx *exec_ctx,
     s->cbs = cronet_bidirectional_stream_create(s->curr_ct.engine, s->curr_gs,
                                                 &cronet_callbacks);
     CRONET_LOG(GPR_DEBUG, "%p = cronet_bidirectional_stream_create()", s->cbs);
-    char *url;
-    const char *method;
+    char *url = NULL;
+    const char *method = NULL;
     s->header_array.headers = NULL;
     convert_metadata_to_cronet_headers(
         stream_op->send_initial_metadata->list.head, s->curr_ct.host, &url,

